articles/ai-services/openai/how-to/on-your-data-configuration.md

```diff
@@ -268,7 +260,7 @@ So far you have already setup each resource work independently. Next you need to
 | `Search Index Data Reader` | Azure OpenAI | Azure AI Search | Inference service queries the data from the index. |
 | `Search Service Contributor` | Azure OpenAI | Azure AI Search | Inference service queries the index schema for auto fields mapping. Data ingestion service creates index, data sources, skill set, indexer, and queries the indexer status. |
 | `Storage Blob Data Contributor` | Azure OpenAI | Storage Account | Reads from the input container, and writes the preprocessed result to the output container. |
-| `Cognitive Services OpenAI Contributor` | Azure AI Search | Azure OpenAI | Custom skill. |
+| `Cognitive Services OpenAI Contributor` | Azure AI Search | Azure OpenAI | To allow Azure AI Search resource access the AOAI embedding endpoint . |
```

Comment:
should read 'to allow azure ai search resource access to the AOAI embedding endpoint'; or 'to allow azure ai search resource to access the aoai embedding endpoint'.